pkg/cli/cmds/server.go

```diff
@@ -280,6 +284,21 @@ func NewServerCommand(action func(*cli.Context) error) cli.Command {
 				Usage:       "(components) Disable " + version.Program + " default network policy controller",
 				Destination: &ServerConfig.DisableNPC,
 			},
+			cli.BoolFlag{
+				Name:        "disable-api-server",
+				Usage:       "(components) Disable running api server",
+				Destination: &ServerConfig.DisableAPIServer,
+			},
+			cli.BoolFlag{
+				Name:        "disable-controller-manager",
+				Usage:       "(components) Disable running kube-controller-manager",
```

Comment:
We call it kube-controller-manager for `kube-controller-manager-args` so we should do the same here.
```suggestion
				Name:        "disable-kube-controller-manager",
				Usage:       "(components) Disable running kube-controller-manager process",
```

Reply:
yeah but I found that we use disable-scheduler for the scheduler disabling originally and for cloud controller manager, so we will have to convert those as well?

Reply:
If you wanted to fix those and keep the original inconsistently-named flags around as deprecated, I would support that. Or you could just leave them as is and we can fix it in a different PR. I'm just in favor of not adding any additional inconsistency.